docs/deployment.rst

```diff
@@ -48,11 +48,29 @@ your own Firefox Accounts OAuth Client for your app domain.
 #. Sign in at https://moz-dev-dash-username.herokuapp.com/ with a Firefox
    Account.
 
+Enable Untappd Auth on Heroku
+--------------------------------------
+
+To enable Untappd sign-ins on your heroku app, you will need to create
+your own Untappd OAuth Client for your app domain.
+
+#. Go to `register your own Untappd OAuth Client`_:
+
+    * Client name: moz-dev-dash-username
+    * Redirect URI: https://moz-dev-dash-username.herokuapp.com/accounts/untappd/login/callback/
+
+   Be sure to copy the client secret - you can't see it again.
+
+#. Go to https://moz-dev-dash-username.herokuapp.com/admin/socialaccount/socialapp/add/
+   to :ref:`enable Untappd Auth` like a local machine; this time using your own new untappd OAuth Client ID and Secret
+
+#. Sign in at https://moz-dev-dash-username.herokuapp.com/ with a Untappd
+   Account.
 
 .. _12-factor app philosophy: http://12factor.net/
 .. _heroku toolbelt: https://toolbelt.heroku.com/
 .. _Create a heroku remote: https://devcenter.heroku.com/articles/git#creating-a-heroku-remote
-.. _register your own Firefox Accounts OAuth Client: https://oauth-stable.dev.lcip.org/console/client/register
+.. _register your own Untappd OAuth Client: https://untappd.com/api/register
```

Comment:
In fact, maybe we shouldn't document the untappd thing at all? That way it's more secret/hidden.